src/core/toga/sources/base.py

```diff
@@ -0,0 +1,71 @@
+
+
+class Datum:
```

Comment:
nit: the name `Datum` bugs me. Personal preference, happy to be overriden, but I would prefer literally anything else.

Reply:
Heh - I like the word Datum :-)

That said - (a) this isn't a name that needs to be exposed in public APIs, and (b) I'm not bound to the name.

Reply:
No - I was aiming at the Latin/English. Datum is the singular form of Data - one datum, many data. However, the linguistic confusion is as good a reason as any to change the name. ``Value``, perhaps?